shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -218,7 +218,6 @@ - (instancetype)initWithName:(NSString*)labelPrefix
 
   _pluginPublications = [[NSMutableDictionary alloc] init];
   _registrars = [[NSMutableDictionary alloc] init];
-  [self recreatePlatformViewController];
```

Comment:
Dont know why this was here, the engine/shell isn't set up yet and it gets recreated when the engine/shell is.